Felienne/hedy
# [BUG FIX] Programs age is displayed correctly again

**Description**
In this PR we remove a weird YAML key only containing a "-" that made it appear like the shown age of a program was a negative value. Not sure why and how this key only contains the "-" value. For some languages it does contain an actual word, but not for either English or Dutch. We remove it in this PR, but that might not be the best way to go.

@Felienne @Mark-Giesen might did have something to do with the changing in the translations? I can't remember the purpose of this "ago-1" key. This code is from before my time😁. 


**Fixes**
This PR fixes #2236

**How to test**
Navigate to your program page and verify that the unclear "-" symbol is removed from the "last edited" value.
